langlib/lang.int/src/main/ballerina/int.bal

```diff
@@ -90,6 +97,16 @@ public isolated function abs(int n) returns int = @java:Method {
 
 # Returns sum of zero or more int values.
 #
+# ```ballerina
+# int:sum(10, 20, 30, 40) ⇒ 100
+# 
+# int[] marks = [50, 65, 78, 95];
+# int total = int:sum(...marks) ⇒ 288
```

Comment:
```suggestion
# int:sum(...marks) ⇒ 288
```

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-lang/pull/38917/commits/015f68edf3c173d04f494b82ab8c01e964211d74